src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -387,4 +403,44 @@ TEST_CASE("request_rate_sequence")
   trrm.TestSequences();
 }
 
+TEST_CASE("request_rate_streaming: test that streaming-specific logic works")
+{
+  PerfAnalyzerParameters params{};
+  params.streaming = true;
+
+  std::shared_ptr<TestRequestRateManager::ThreadStat> thread_stat{
+      std::make_shared<TestRequestRateManager::ThreadStat>()};
+  std::shared_ptr<TestRequestRateManager::ThreadConfig> thread_config{
+      std::make_shared<TestRequestRateManager::ThreadConfig>(0, 0)};
+
+  SUBCASE("enable_stats true")
+  {
+    TestRequestRateManager trrm(params);
+    trrm.schedule_.push_back(std::chrono::nanoseconds(1));
+
+    std::future<void> infer_future{std::async(
+        &TestRequestRateManager::Infer, &trrm, thread_stat, thread_config)};
+
+    early_exit = true;
+    infer_future.get();
+
+    CHECK(trrm.stats_->start_stream_enable_stats_value == true);
+  }
+
+  SUBCASE("enable_stats false")
+  {
+    TestRequestRateManager trrm(
+        params, false /* is_decoupled */, true /* is_sequence */);
```

Comment:
comment is incorrect. first is is_sequence

Reply:
Fixed 👍